src/query/atomWithInfiniteQuery.ts

```diff
@@ -88,6 +89,15 @@ export function atomWithInfiniteQuery<
           | { data?: undefined; error: TError }
       ) => {
         if (result.error) {
+          if (isCancelledError(result.error)) {
+            // fetch has been canceled,
+            // revert to prevData
```

Comment:
```suggestion
            // fetch has been canceled, revert to prevData
```
unless prettier complains.

Reply:
I removed the comments